modules/engine/src/listeners.ts

```diff
@@ -1024,6 +1024,7 @@ export const isWithdrawTransfer = async (
   chainAddresses: ChainAddresses,
   chainService: IVectorChainReader,
 ): Promise<Result<boolean, ChainError>> => {
+  // TODO: cache this!
```

Comment:
This is cached in the `chainReader` class

Reply:
Oh sick!